web/style/cart.scss

```diff
@@ -0,0 +1,137 @@
+@import "_variables.scss";
+@import "common.scss";
+
+.cart-container {
+  // height: 100vh;
+  // @extend .flex-column;
+  background-color: #eee;
+
+  &--head {
+    @extend .flex-column;
+    background-color: white;
+    padding: 20px;
+
+    &_text {
+      font-size: 18px;
+    }
+  }
+ 
+  .cart-item {
+    @extend .flex-row;
+    margin-top: 10px;
+    background-color: #fff;
+    &__img {
+      width: 20%;
+      height: 21%;
+    }
+  
+  }
+
+  
+.cart-item-detail {
+  @extend .flex-column;
+  &__heading {
+      font-size: 1.5rem;
+      padding: 15px;
+  }
+  &__info {
+      @extend .flex-row;
+      font-size: 1rem;
+      align-items: center;
+      
+      &--count {
+          @extend .flex-row;
+      }
+      &--item{
+        &-count{
+          margin-top: 4px;
+        }
+      }
+      &--unitPrice {
+        margin-top: 4px;
+        
+      }
+      &--symbol {
+          margin-top: 2px;
+          font-size: 24px;
+
+      }
+      &--remove,
+      &--add {
+        @include break-point(desktop) {
+          margin-right: 25px;
+          margin-left: 25px;
+          height: 25px;
+          width: 25px;
+        }
+        @include break-point(tablet) {
+          margin-right: 20px;
+          margin-left: 20px;
+          height: 25px;
+          width: 25px;
+        }
+          margin-right: 12px;
+          margin-left: 12px;
+          height: 25px;
+          width: 25px;
+          background: $background-color-secondary;
+          text-align: center;
+          color: $text-color-secondary;
+          border-radius: 100%;
+          display: inline-flex;
+          flex-direction: column;
+          justify-content: center;
+          align-items: center;
+          font-size: 1rem;
+          font-weight: bold;
+          text-decoration: none;
+          font-family: sans-serif;
+          margin-bottom: 7px;
+      }
+      &--total {
+          margin-right: 50px;
+          margin-bottom: 10px;
+      }
+  }
+}
+
+.cart__content__price {
+  margin-top: 16px;
+  width: 100%;
+  padding: 12px;
+  display: flex;

```

Comment:
You gad created a flex based re-usable classes why you are not applying those and defining flex at every place?